### DISCUSSION TITLE
Rolladen Steuerung Qubino ZMNHCD

### DISCUSSION THREAD

**Discussion by mericon** [Zefau/ioBroker.jarvis#938] [Q&A]
*Please note that you may restore an old layout via jarvis adapter configuration directly within ioBroker

**Prerequisites / Voraussetzungen**
- [ ] Which jarvis version are you running / Welche Version von jarvis nutzt du? **BITTE GENAUE ANGABE!** 2.2.0
- [ ] Fehler / Screenshot der Browser Konsole (F12), siehe [Fehleranalyse und -Meldung](https://github.com/Zefau/ioBroker.jarvis/wiki/de-Errors)

**Describe the bug / Beschreibe den Fehler**
Ich habe für meine Rolladen die zwave Module Qubino ZMNHCD
Diese bieten zur Aktion nur die Datenpunkte Up, down und target value.
Wenn ich nun in Jarvis diese als blinds hinzufügen will bekomme ich nur die Auswahl Activity und stop, wie kann ich aus dem Stop ein Up oder down machen?

**Screenshots**
Sofern möglich, bitte Screenshots einfügen, um den Fehler bildlich zu erläutern.

____

**Please note that bug reports may be closed if the information above are not given!**

**Bitte beachte, dass Bug-Reports ggf. einfach geschlossen werden, wenn die oben genannten Kriterien nicht angemessen gegeben sind!**

**Comment by mericon** [comment]
Okay nach viel testen habe ich nun targetvalue als Activity genommen. Nur stoppen kann ich nicht da es hierfür keinen Datenpunkt gibt. Lässt sich das irgendwie lösen?

**Reply by Zefau** [reply]
Mach mal bitte zum Verständnis einen Screenshot der Datenpunkt-Struktur und der aktuellen Einstellungen des Geräts in jarvis.

**Comment by mericon** [comment]
Hi,

ich habe das problem im moment durch einen etwas unschönen workaround umgangen.
Anbei bilder von den "Werksdatenpunken", von meinen Jarvis einstellungen und meinem workaround.

Zum umgehen des problems habe ich einen userdatenpunkt "user.0.Rolladen.stop" angelegt und diesen der Stoptaste zugewiesen, sobald diese betätigt wird werden alle UP und Down datenpunkte meiner rollos auf false gesetzt. Nicht schön aber aktuell funktionsfähig. Wenn man das ohne große Arbeit schöner machen kann bin ich natürlich nicht abgeneigt (Aktuell suche ich auch nach einer lösung wie ich die Rolladen mit dem Datenpunkt "Target" und einem Slider fahren kann.)
![jarvis](https://user-images.githubusercontent.com/1723113/131914745-8a79403b-4885-4f3e-86c3-ab7ba53116a9.PNG)
![nodes](https://user-images.githubusercontent.com/1723113/131914747-dd132a76-08f3-4d98-aa07-97e1bfc7dd9e.png)
![stop-workaround1](https://user-images.githubusercontent.com/1723113/131915046-70fce58b-2c86-45fb-a6ae-6e7bcbdbbc70.PNG)
![stop-workaround2](https://user-images.githubusercontent.com/1723113/131915048-07ca728c-4f17-4b99-95a5-6a41fe7ca697.PNG)
